spec/lib/models/order_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+module BinanceClient
+  RSpec.describe Order do
+    subject(:order) do
+      described_class.new({
+        symbol: "ETHUSDT",
+        executed_quantity: "1.20000000",
+        cummulative_quote_quantity: "2769.10800000",
+        order_id: 8848559626,
+        side: "BUY",
+      })
+    end
+
+    its(:symbol) { is_expected.to eq "ETHUSDT" }
+    its(:executed_quantity) { is_expected.to eq "1.20000000" }
```

Comment:
Maybe you're still working on it, but what I meant by typecasting is:
```suggestion
    its(:executed_quantity) { is_expected.to eq 1.2 }
```